File: FrontEnd/src/app/features/billing/invoices/classes/view-models/pdf/invoice-pdf-vm.ts
import { InvoicePdfPartyTypeVM } from './invoice-pdf-partyType-vm'
import { InvoicePdfHeaderVM } from './invoice-pdf-header-vm'
import { InvoicePdfSummaryVM } from './invoice-pdf-summary-vm'
import { InvoicePdfAadeVM } from './invoice-pdf-aade-vm'

export interface InvoicePdfVM {

    header: InvoicePdfHeaderVM
    issuer: InvoicePdfPartyTypeVM
    counterPart: InvoicePdfPartyTypeVM
    summary: InvoicePdfSummaryVM
    aade: InvoicePdfAadeVM

}

File: API/Features/RetailSales/ViewModels/Pdf/InvoicePdfVM.cs
using System.Collections.Generic;
using API.Infrastructure.Classes;

namespace API.Features.RetailSales {

    public class InvoicePdfVM {

        public string ReservationId { get; set; }
        public InvoicePdfPartyVM Issuer { get; set; }
        public InvoicePdfDocumentTypeVM DocumentType { get; set; }
        public InvoicePdfHeaderVM Header { get; set; }
        public InvoicePdfReservationVM Reservation { get; set; }
        public InvoicePdfPaxAndPricesVM Persons { get; set; }
        public List<InvoicePdfPassengerVM> Passengers { get; set; }
        public InvoicePdfAadeVM Aade { get; set; }
        public InvoicePdfSummaryVM Summary { get; set; }
        public SimpleEntity[] BankAccounts { get; set; }

    }

}